src/elements/container/container/container.visual.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { SbbBreakpointUltraMin } from '@sbb-esta/lyne-design-tokens';
+import { setViewport } from '@web/test-runner-commands';
+import { html, type TemplateResult } from 'lit';
+
+import { describeViewports, visualDiffDefault } from '../../core/testing/private.js';
+
+import './container.js';
+import '../../title.js';
+import '../../button.js';
+
+describe(`sbb-container`, () => {
+  const colorCases = ['transparent', 'white', 'milk'];
+
+  const backgroundExpandedCases = [false, true];
+
+  const containerContent = (): TemplateResult => html`
+    <sbb-title level="4">Example title</sbb-title>
+    <p class="sbb-text-s">The container component will give its content the correct spacing.</p>
+    <p class="sbb-text-s">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+      labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+      laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+      voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+    </p>
+    <sbb-secondary-button style="margin-block-end: 3rem;">See more</sbb-secondary-button>
+  `;
+
+  describeViewports({ viewportHeight: 600 }, () => {
+    for (const color of colorCases) {
+      it(
+        `color=${color}`,
+        visualDiffDefault.with(async (setup) => {
+          await setup.withFixture(
+            html` <sbb-container color=${color}> ${containerContent()} </sbb-container> `,
+            { backgroundColor: 'var(--sbb-color-silver)' },
```

Comment:
padding should be set to zero for all test cases of the container (even the not expanded)

Reply:
I meant to leave the padding because it helps to have a contrast with the background color. Now I have a clear visual separation between the component colors (particularly handy with the transparent variant)

Reply:
Hmm, the problem I see is, that from a design perspective, it's only allowed to use the container (and sticky bar) full width. So if we have the padding here, it's not showing how it's supposed to be used (even if it's only for our internal use). The part with the background color should also work if the padding is 0. Or which exact case cannot be tested?

Reply:
Makes sense.
The background thing was just a convenience factor, it works without the padding too